### PR TITLE
dist/openshift: migrate graph-builder to toml config

### PR DESCRIPTION
**Pull request by steveej**
This will allow passing in different plugin configuration, which we may
require for e2e tests at one point.

- [x] Depends on #242.

### CODE REVIEW

File: hack/e2e.sh [hunk 1]
```diff
@@ -33,11 +33,30 @@ oc secrets link default ci-pull-secret --for=pull
 oc new-app -f template/cincinnati.yaml \
   -p IMAGE="${IMAGE_FORMAT%/*}/stable" \
   -p IMAGE_TAG=deploy \
-  -p GB_CINCINNATI_REPO="redhat/openshift-cincinnati-test-public-manual" \
   -p GB_CPU_REQUEST=50m \
   -p PE_CPU_REQUEST=50m \
   -p RUST_BACKTRACE="1" \
-  -p GB_REGISTRY_CREDENTIALS_PATH="" \
+  -p GB_PLUGIN_SETTINGS='
```
Comment (vrutkovs) [inline hack/e2e.sh:39]: nitpick: params are growing large, lets put these in a file and use `oc new-app --param-file`?

Comment (steveej) [reply hack/e2e.sh:39]: It's a bit hard to justify, given that we're talking about code in *hack*. We could move it to dist and take it more serious though.

**Comment by vrutkovs** [comment]
Cluster failed to start

/retest

File: hack/e2e.sh [hunk 2]
```diff
@@ -33,15 +33,39 @@ oc secrets link default ci-pull-secret --for=pull
...
+      name = "node-remove"
+
+      [[plugin_settings]]
+      name = "edge-add-remove"
+  ' \
   ;
 
 # Wait for dc to rollout
-oc wait --for=condition=available --timeout=5m deploymentconfig/cincinnati
+oc wait --for=condition=available --timeout=5m deploymentconfig/cincinnati || {
```
Comment (vrutkovs) [inline hack/e2e.sh:58]: Excellent, thanks for doing that! I think we also want `oc describe deploymentconfig/cincinnati` and `oc get events` here.

The only issue is that the objects in the template may eventually diverge and we won't find that out until tests start to fail

Comment (steveej) [reply hack/e2e.sh:58]: Do you care about the order of these?

File: hack/e2e.sh (hunk 1 above)
Comment (steveej) [reply hack/e2e.sh:39]: Do you consider this a blocker?

Comment (vrutkovs) [reply hack/e2e.sh:39]: No, lets rework this later

File: hack/e2e.sh (hunk 2 above)
Comment (vrutkovs) [reply hack/e2e.sh:58]: The order doesn't matter here really

**Comment by vrutkovs** [comment]
some odd CI flake

/retest

**Review by vrutkovs — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**